doc/build.rst

```diff
@@ -160,9 +159,9 @@ The Python package is located at ``python-package/``.
 
 1. Install system-wide, which requires root permission:
 
-.. code-block:: bash
+   .. code-block:: bash
```

Comment:
Can you fix the indentation alignment here? The `..` on this line should be aligned with the `..` on line 166.

Reply:
Actually, that was deliberate. Otherwise, the code block appears outside the bullet.